### PR TITLE
Cleanup wp-taxonomy-filter

### PR DESCRIPTION
**Pull request by cristirusu**
Cleanup wp-taxonomy-filter

### CODE REVIEW

File: lib/wp-taxonomy-filter/class-cf-taxonomy-filter.php [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+			self::date_filter( $start_options, $end_options );
+		}
+
+		self::submit_button( $this->options['submit_options'] );
+
+		self::end_form();
+	}
+
+	/**
+	 * Echo a date range filter form element
```
Comment (mbucurcf) [inline lib/wp-taxonomy-filter/class-cf-taxonomy-filter.php]: Missing . at the end of the line.

File: lib/wp-taxonomy-filter/class-cf-taxonomy-filter.php [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+	 * @return void
+	 */
+	public function build_form() {
+		self::start_form( $this->options['form_options'] );
+
+		if ( ! empty( $this->options['taxonomies'] ) ) {
+			foreach ( $this->options['taxonomies'] as $taxonomy => $args ) {
+				if ( is_array( $args ) ) {
+					self::tax_filter( $taxonomy, $args );
+				} // Just passed in taxonomy name with no options
```
Comment (mbucurcf) [inline lib/wp-taxonomy-filter/class-cf-taxonomy-filter.php]: 1. Missing . at the end of the line.
2. Is that in a valid location?

File: lib/wp-taxonomy-filter/class-cf-taxonomy-filter.php [hunk 3]
```diff
@@ -0,0 +1,426 @@
...
+			'class'  => '',
+			'action' => home_url( '?s=' ),
+		);
+
+		$args = array_merge( $defaults, $args );
+		// Used in js for URL cleanup.
+		$args = self::add_class( 'cftf-filter', $args );
+
+		echo '
+<form method="GET"' . wp_kses( self::build_attrib_string( $args ), array() ) . '>';
```
Comment (mbucurcf) [inline lib/wp-taxonomy-filter/class-cf-taxonomy-filter.php]: I know it's intended to look OK as HTML, but it looks weird as PHP.

File: lib/wp-taxonomy-filter/class-cf-taxonomy-filter.php [hunk 4]
```diff
@@ -0,0 +1,426 @@
...
+	}
+
+	/**
+	 * Closes the form and adds the action.
+	 *
+	 * @return void
+	 **/
+	public static function end_form() {
+		echo '
```
Comment (mbucurcf) [inline lib/wp-taxonomy-filter/class-cf-taxonomy-filter.php]: I know it's intended to look OK as HTML, but it looks weird as PHP.

File: lib/wp-taxonomy-filter/class-cf-taxonomy-filter.php (hunk 1 above)
Comment (cristirusu) [reply lib/wp-taxonomy-filter/class-cf-taxonomy-filter.php]: Updating. However latest phpcs does not report this warning for method descriptions comments

Comment (mbucurcf) [reply lib/wp-taxonomy-filter/class-cf-taxonomy-filter.php]: Oh, I see. Still nice to be consistent with inline comments, though :) But not a biggie...

**Pull request merged**